Translators/WZDx/main.py

```diff
@@ -0,0 +1,84 @@
+import functions_framework
+import json
+import copy
+import logging
+import os
+from Translators.WZDx.request_wrapper import get_rsu_request, get_sdw_request

```

Comment:
I am not sure how this will eventually be run, but to run this locally I had to change all of these imports to be relative (just from request_wrapper import ...)

Reply:
Changed to relative imports.